server/src/models/configModel.ts

```diff
@@ -23,4 +36,10 @@ export interface ClientConfigModel {
   waitingTitle: string;
   waitingSubtitle: string;
   logoUrl: string;
+  postCall?: {
+    survey?: {
+      type: PostCallSurveyType;
+      options: MSFormsSurveryOptions | ThirdPartySurveyOptions;
+    };
+  };
```

Comment:
This code is repeated. Can we extract this to an interface so we can use the interface in both places?

Reply:
This change is applied. Thanks!